Header header
float64 max_weight